hash/psx.xml

```diff
@@ -370,36 +376,65 @@ For an overview of US PS1 discs that have not been dumped and added to redump's
 		</part>
 	</software>
 
-	<software name="3xtreme">
-		<!-- Original images (Redump)
+	<software name="3xtreme" supported="partial">
+		<!--
+		http://redump.org/disc/2715/
+		<rom name="3Xtreme (USA).cue" size="196" crc="a689937f" sha1="6d16c06c59a0eab13c1364c03dce6db699c5793a"/>
 		<rom name="3Xtreme (USA) (Track 1).bin" size="340649568" crc="c16782c6" sha1="4e6ced10157eda5e1881e95d1bd8363ae6b560f8"/>
 		<rom name="3Xtreme (USA) (Track 2).bin" size="26732832" crc="efb07e5e" sha1="60da792ac141a78f7fe875788d0e73b5bc1b943d"/>
 		-->
 		<description>3Xtreme (USA)</description>
 		<year>1999</year>
-		<publisher>989 Studios</publisher>
+		<publisher>989 Sports</publisher>
+		<notes>Compatibility set as PAL since it didn't work on NTSC system</notes>
+		<info name="developer" value="989 Sports" />
+		<info name="language" value="English" />
 		<info name="serial" value="SCUS-94231"/>
+		<sharedfeat name="compatibility" value="PAL-E"/>
```

Comment:
This doesn't really make sense. At worst you should just drop the `sharedfeat` instead, it clearly sounds as something that needs investigating (and inclined to `supported="no"` here).

Reply:
Changed flag to supported="no" and change to compatibility to NTSC-U